### PR TITLE
Add Default Image

### PR DESCRIPTION
**Pull request by manav809**
I added A default image for the resturants that don't have an image. 
Screen Shot 2023-04-07 at 8 20 54 PM

### CODE REVIEW

**Review by Remos96 — DISMISSED** [review]
Looks good but add description to PR

**Comment by JoshuaRamc** [comment]
Add description to PR, and also check with Devin if we want to leave comments in code

File: client/src/pages/Search/GridView/RestaurantCard/ImageCard.js [hunk 1]
```diff
@@ -7,7 +7,15 @@ const classes = {
...
 function ImageCard({ src }) {
-  return <img src={src} style={classes.imageCard} alt={src} />;
+  return <img 
+  src={src}
+  style={classes.imageCard}
+  onError={({ currentTarget }) => {
+    currentTarget.onerror = null; // prevents looping
+    currentTarget.src="https://images.unsplash.com/photo-1546069901-ba9599a7e63c?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxleHBsb3JlLWZlZWR8MXx8fGVufDB8fHx8&w=1000&q=80";
+  }}
+  alt={src}
```
Comment (DLi7077) [inline client/src/pages/Search/GridView/RestaurantCard/ImageCard.js:17]: alt should describe the image for screen readers, like "restaurant" or "dolphin eating cheese"

**Comment by JoshuaRamc** [comment]
Also I would say do not put the image as a link, rather add it the same way we added the budget eats logo

File: client/src/pages/Search/GridView/RestaurantCard/ImageCard.js [hunk 2]
```diff
@@ -7,7 +7,15 @@ const classes = {
 };
 
 function ImageCard({ src }) {
-  return <img src={src} style={classes.imageCard} alt={src} />;
+  return <img 
+  src={src}
+  style={classes.imageCard}
+  onError={({ currentTarget }) => {
+    currentTarget.onerror = null; // prevents looping
+    currentTarget.src="https://images.unsplash.com/photo-1546069901-ba9599a7e63c?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxleHBsb3JlLWZlZWR8MXx8fGVufDB8fHx8&w=1000&q=80";
```
Comment (DLi7077) [inline client/src/pages/Search/GridView/RestaurantCard/ImageCard.js:15]: Not a fan of how this was implemented.
You could just have something like:
```jsx
src = { restaurant_image ?? fallback_image }
```
Ask @JoshuaRamc about value coalescing 

Also, don't like how the default picture is a colorful chicken salad. Doesn't give the impression that the restaurant has no image. Use a bland image like this instead:
image

**Review by DLi7077 — CHANGES_REQUESTED** [review]
Please format your code before commiting

File: client/src/pages/Search/GridView/RestaurantCard/ImageCard.js (hunk 2 above)
Comment (Remos96) [inline client/src/pages/Search/GridView/RestaurantCard/ImageCard.js:15]: Remove link, download the image

Comment (DLi7077) [reply client/src/pages/Search/GridView/RestaurantCard/ImageCard.js:15]: using nullish coalescing lets you remove that `onError` prop

Comment (DLi7077) [reply client/src/pages/Search/GridView/RestaurantCard/ImageCard.js:15]: hint: This file should only have 2 line changes

import defaultImage
nullish coalesce on image url

File: client/src/pages/Search/GridView/RestaurantCard/ImageCard.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import default_restaurant_img from "../../../../assets/images/default-restaurant-img.png";
```
Comment (DLi7077) [inline client/src/pages/Search/GridView/RestaurantCard/ImageCard.js:1]: Nit use camelCase

**Review by DLi7077 — CHANGES_REQUESTED** [review]
One last change.
Squash your changes and we're good to go

**Comment by manav809** [comment]
> One last change. Squash your changes and we're good to go

I squashed

**Review by DLi7077 — APPROVED** [review]

**Review by JoshuaRamc — APPROVED** [review]
coalesce operator logic looks good

**Pull request merged**